Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM alpine:3 AS builder
```

Comment:
Better start from `ruby` container

Reply:
Removed build part from container in 9c8f130